freyja/utils.py

```diff
@@ -898,12 +898,6 @@ def collapse_barcodes(df_barcodes, df_depth, depthcutoff, locDir, output):
         return df_barcodes
 
     # load lineage data
```

Comment:
I think we should probably keep the `lineages.yml` load here, as per above comment.